lib/utapi/utilities.js

```diff
@@ -103,9 +102,10 @@ export function listMetrics(metricType) {
         .option('-p, --port <port>', 'Port of the server')
         .option('--ssl', 'Enable ssl')
         .option('-v, --verbose')
+        .option('-r, --recent', 'List metrics of the latest 15 minute interval')
```

Comment:
Updated to: 'List metrics including the previous and current 15 minute interval'